seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rocketmq-e2e/src/test/java/org/apache/seatunnel/e2e/connector/rocketmq/RocketMqIT.java

```diff
@@ -214,6 +214,21 @@ public void testSourceRocketMqTextToConsole(TestContainer container)
         Assertions.assertEquals(0, execResult.getExitCode(), execResult.getStderr());
     }
 
+    @TestTemplate
+    public void testStreamingSourceRocketMqTextToConsole(TestContainer container)
```

Comment:
Thanks @cqutwangyu for updated the test case! There are still some code should be changed. Because we create an streaming job. So I think it will never return exit code if executed normally. I think you should async run `executeJob`, then sleep main thread more than 60s. Finally check the job still running.
eg:
```java
CompletableFuture future = CompletableFuture.supplyAsync(
        () -> {
            try {
                container.executeJob("/rocketmq-source_streaming_text_to_console.conf");
            } catch (Exception e) {
                log.error("Commit task exception :" + e.getMessage());
                throw new RuntimeException(e);
            }
            return null;
        });
Thread.sleep(60 * 1000);
// meaning job still running
Assertions.assertFalse(future.isDone());
```

Reply:
Okay, thanks for the guidance. I found that the 'Seek offset illegal' error does not cause the FlinkEngine program to exit, but instead triggers exception logging on a loop. But in ` org. Apache. Seatunnel. Example. Engine. SeaTunnelEngineExample ` can make the program exits.

Reply:
In the 'e2e' test case, the following error message appears when running 'SeaTunnelContainer'

```
109280 [main] INFO  org.apache.seatunnel.e2e.common.junit.TestLoggerExtension - 
================================================================================
Test org.apache.seatunnel.e2e.connector.rocketmq.RocketMqIT.testSourceRocketMqTextToConsole is running.
--------------------------------------------------------------------------------
109281 [main] INFO  🐳 [openjdk:8] - Creating container for image: openjdk:8
110577 [main] INFO  🐳 [openjdk:8] - Container openjdk:8 is starting: 46b64848baaa97a5b6338459977e374aa2f2cf78cc5f0f1e28dfbffb458bcc40
111145 [docker-java-stream--684067844] INFO  🐳 [seatunnel-engine:openjdk:8] - STDERR: Error: Could not find or load main class 
111154 [testcontainers-wait-2] WARN  org.testcontainers.containers.wait.internal.InternalCommandPortListeningCheck - An exception while executing the internal check: Container.ExecResult(exitCode=137, stdout=, stderr=/bin/sh: 1: nc: not found
/bin/bash: connect: Cannot assign requested address
/bin/bash: line 1: /dev/tcp/localhost/5801: Cannot assign requested address
)
111154 [main] INFO  🐳 [openjdk:8] - Container openjdk:8 started in PT1.873S
111154 [main] INFO  org.apache.seatunnel.e2e.common.junit.TestCaseInvocationContextProvider - The TestContainer[SeaTunnel:2.3.1] is running.
111505 [ForkJoinPool.commonPool-worker-2] INFO  org.apache.seatunnel.e2e.common.container.seatunnel.SeaTunnelContainer - test in container: SeaTunnel:2.3.1
111949 [ForkJoinPool.commonPool-worker-2] INFO  org.apache.seatunnel.e2e.common.container.AbstractTestContainer - Execute config file: \tmp\rocketmq-source_text_streaming_to_console.conf to Container[openjdk:8] 
==================== Shell Command start ====================
/tmp/seatunnel/bin/seatunnel.sh --config /tmp/rocketmq-source_text_streaming_to_console.conf
==================== Shell Command end   ====================
111973 [ForkJoinPool.commonPool-worker-2] ERROR org.apache.seatunnel.e2e.connector.rocketmq.RocketMqIT - Commit task exception :Status 409: {"message":"Container 46b64848baaa97a5b6338459977e374aa2f2cf78cc5f0f1e28dfbffb458bcc40 is not running"}

131525 [NettyClientSelector_1] INFO  RocketmqRemoting - closeChannel: close the connection to remote address[] result: true
161519 [NettyClientSelector_1] INFO  RocketmqRemoting - closeChannel: close the connection to remote address[] result: true

```